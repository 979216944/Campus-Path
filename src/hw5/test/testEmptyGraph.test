# Tests that a graph can be created(empty graph).

# Create a graph
CreateGraph g

# Print the nodes in the graph
ListNodes g