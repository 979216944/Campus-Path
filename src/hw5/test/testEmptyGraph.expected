# Tests that a graph can be created(empty graph).

# Create a graph
created graph g

# Print the nodes in the graph
g contains: